delta_backend/tests/test_delta.py

```diff
@@ -187,38 +199,38 @@ def test_handler_success_delete_logical(self, mock_boto_resource):
         self.assertEqual(put_item_data["Operation"], Operation.DELETE_LOGICAL)
         self.assertEqual(put_item_data["ImmsID"], imms_id)
 
+
     @patch("boto3.resource")
     @patch("boto3.client")
-    def test_handler_exception_intrusion_check(self, mock_boto_resource, mock_boto_client):
+    def test_handler_exception_intrusion_check(self, mock_boto_client, mock_boto_resource):
```

Comment:
Can we remove reference to "intrusion check" now we've removed it from the main code? No idea what that was referring to 😂